recipes-multimedia/qtivi-mopidy-plugin/qtivi-mopidy-plugin_git.bb

```diff
@@ -12,8 +12,6 @@ SRCREV = "18198a616501bca072689a31c1581c879605da68"
 
 S = "${WORKDIR}/git"
 
-inherit qmake5
```

Comment:
Does removing this affect neptune images?

Reply:
No, it builds fine.